cmd/image-inspector.go

```diff
@@ -16,7 +16,7 @@ func main() {
 	flag.StringVar(&inspectorOptions.DstPath, "path", inspectorOptions.DstPath, "Destination path for the image files")
 	flag.StringVar(&inspectorOptions.Serve, "serve", inspectorOptions.Serve, "Host and port where to serve the image with webdav")
 	flag.BoolVar(&inspectorOptions.Chroot, "chroot", inspectorOptions.Chroot, "Change root when serving the image with webdav")
-	flag.StringVar(&inspectorOptions.DockerCfg, "dockercfg", inspectorOptions.DockerCfg, "Location of the docker configuration file")
+	flag.StringVar(&inspectorOptions.DockerCfg, "dockercfg", inspectorOptions.DockerCfg, "Location of the docker configuration files seperated by ',' if more than one")
```

Comment:
@enoodle what about files with a comma `,` in their name? :smile: 

I think there should be a way to specify more than once the same flag, e.g.:

`image-inspector ... --dockercfg=config1 --dockercfg=config2 --dockercfg=config3 ...`

cc @pweil-